crates/flightrepl/src/lib.rs

```diff
@@ -38,7 +38,13 @@ pub async fn run(repl_config: ReplConfig) -> Result<(), Box<dyn std::error::Erro
     let channel = Channel::from_shared(repl_config.repl_flight_endpoint)?
         .connect()
         .await?;
-    let mut client = FlightServiceClient::new(channel);
+
+    // The encoder/decoder size is limited to be 100MB by default.
+    // This may not be enough for large queries.
+    // In future, paging will be supported to handle the large query output.
+    let mut client = FlightServiceClient::new(channel)
```

Comment:
Let's allow the user to override this with a flag?

Reply:
thought about this but decided not to go with this option. Once the `spice sql` is more feature complete to support paging like `psql`, user won't need to worry about this decoder/encoder size. 

The issue is described here: https://github.com/spiceai/spiceai/issues/874 which is sitting between feature/bug.

Even allowing user to config this, it won't be usable as it will flooding the terminal with super long content.

Reply:
And this PR is only an improvement to current situation which gives no error about what is happening

Reply:
Yeah I'm not sure how useful it is to even have 4MB of data flooding the screen here. Only use-case I think raising this limit makes sense for is if we would save the results to a parquet file or something.

Reply:
But Mitch has a point that the error message says "configured maximum size" but we don't allow them to configure it.

Reply:
yeah I will modify the error message.